cli/tests/environment-init.bats

```diff
@@ -160,6 +160,29 @@ function check_with_dir() {
   assert_equal "$init_system" "$NIX_SYSTEM"
 }
 
+@test "'flox init' sets up a working Python environment using requirements.txt" {
+  OWNER="owner"
+  NAME="name"
+
+  echo "requests" > requirements.txt
+  "$FLOX_BIN" init --auto-setup --name "$NAME"
+  "$FLOX_BIN" activate -- python -c "import requests"
+
+  floxhub_setup "$OWNER"
+
+  "$FLOX_BIN" push --owner "$OWNER"
+
+  rm -rf .flox
+
+  "$FLOX_BIN" pull "$OWNER/$NAME"
+
+  "$FLOX_BIN" activate -- python -c "import requests"
+
+  rm -rf .flox
+
+  "$FLOX_BIN" activate --trust -r "$OWNER/$NAME" -- python -c "import requests"
+}
```

Comment:
@tomberek this test takes 30 seconds. Not sure if I should move it to functional tests or something, but at least wanted to flag it

Reply:
That seems to be quite a bit. I doubt that is mostly python. Nixpkgs?

Reply:
A build for something we're all is already fetched and a warm eval cache should be nearly free... we need a breakdown of those 30s.